qaframework-legacy-tests/pom.xml

```diff
@@ -60,7 +60,7 @@
 		<dependency>
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-databind</artifactId>
-			<version>2.11.2</version>
```

Comment:
cc @dkayiwa we need to consent with using dependant bot.  i dont think that we will need its upgrades in particular

Reply:
Those upgrades are usually good to have, unless they break the build and you do not have time to fix it.

Reply:
I agree with @dkayiwa ! Let have the tests run against these PR to varify whether the upgrade is safe. Have put up a PR at  https://github.com/openmrs/openmrs-contrib-qaframework/pull/404 to cater for the fix.